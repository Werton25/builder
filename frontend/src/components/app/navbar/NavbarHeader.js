import {Link} from 'react-router-dom';
import React from 'react';

export default class NavbarHeader extends React.Component {
  render() {
    return (
      <div className='navbar-header'>
        <label className='navbar-brand'>
          <Link to='/'>Api</Link>
        </label>
      </div>
    );
  }
}
