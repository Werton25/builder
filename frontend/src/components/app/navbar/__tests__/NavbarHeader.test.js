import React from "react";
import renderer from 'react-test-renderer';
import NavbarHeader from "../NavbarHeader";
import {MemoryRouter} from "react-router-dom";

describe('NavbarHeader tests', () => {
  it('NavbarHeader renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter initialEntries={['/']}>
          <NavbarHeader/>
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
});
