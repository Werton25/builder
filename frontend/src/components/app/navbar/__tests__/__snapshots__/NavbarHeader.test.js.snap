// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarHeader tests NavbarHeader renders correctly 1`] = `
<div
  className="navbar-header"
>
  <label
    className="navbar-brand"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Api
    </a>
  </label>
</div>
`;
